### ISSUE TITLE
Error trying to backup to Google Cloud Storage

### ISSUE THREAD

**Issue by lsansaa** [backup-manager/laravel-4#120]
Trying to run the backup-manager from command with "--destination=gcs" I get the next error:

```
In FilesystemProvider.php line 43:

  The requested filesystem type Gcs is not currently supported.
```
I'm using Lumen 5.8 and php 7.1.3.

No issues running with "destination=local"

**Comment by ShawnMcCool** [comment]
Hi there

Is it possible that you have not installed the necessary driver?

![image](https://user-images.githubusercontent.com/560749/66915610-b8653580-f019-11e9-8e0e-9a67300b4ebf.png)

https://github.com/backup-manager/backup-manager#installation

**Comment by lsansaa** [comment]
Yes, I already tried with both drivers, the "league/flysystem-aws-s3-v3" as the documentation says that should works with both, and "league/flysystem-aws-s3-v2".

We are already trying the backup with another methods, but I'm still want to try with this one.
